readme.md

```diff
@@ -171,6 +171,21 @@ const {body: readableStream} = await fetch('https://example.com');
 console.log(await getStream(readableStream.pipeThrough(textDecoderStream)));
 ```
 
+### JSON streaming
+
+[`getStreamAsArray()`](#getstreamasarraystream-options) can be combined with JSON streaming utilities to parse JSON incrementally.
+
+```js
+import {getStreamAsArray} from 'get-stream';
+import streamJson from 'stream-json';
+import streamJsonArray from 'stream-json/streamers/StreamArray.js';
+
+const stream = fs.createReadStream('big-array-of-objects.json', {encoding: 'utf8'});
+console.log(await getStreamAsArray(
+	stream.pipe(streamJson.parser()).pipe(streamJsonArray.streamArray()),
```

Comment:
`pipeline` is generally recommended instead of pipe methods.

Reply:
That's right! Good catch.

What about `compose()` instead? While `pipeline()` starts consuming the stream and returns a promise, `compose()` does not start consuming it and returns the stream.

Reply:
Yeah, `compose` is probably the right choice. I've been too used to (apparently) incorrectly using `pipeline`.